### ISSUE TITLE
skimage error from skimage.transform

### ISSUE THREAD

**Issue by psteinb** [mertkeser/detr_simple#1]
with python `3.8.5` I get the following error when doing: `python ./main.py`

```
$ python ./main.py 
Traceback (most recent call last):
  File "./main.py", line 6, in <module>
    from nuim_dataloader.nuim_dataloader import NuimDataset, Rescale, transforms, collate_fn_nuim
  File "/p/home/jusers/steinbach1/juwels/projects/detr_simple/nuim_dataloader/nuim_dataloader.py", line 5, in <module>
    from skimage import transform
  File "/p/home/jusers/steinbach1/juwels/projects/detr_simple/local-py38/lib/python3.8/site-packages/skimage/__init__.py", line 94, in <module>
    from ._shared.version_requirements import ensure_python_version
  File "/p/home/jusers/steinbach1/juwels/projects/detr_simple/local-py38/lib/python3.8/site-packages/skimage/_shared/version_requirements.py", line 3, in <module>
    from packaging import version as _version
ModuleNotFoundError: No module named 'packaging'
```

**Comment by mertkeser** [comment]
Hi Peter, 

Could you try "pip install packaging". We did the environment with "python 3.8.0". If the problem still persists could you try it with "python 3.8.0". 

Best regards,
Mert

**Comment by psteinb** [comment]
Right. I’ll put that on the list. Apparently there are some inconsistencies with the `requirements.txt` coming with this repo.
I’ll continue looking and keep you posted.

**Comment by psteinb** [comment]
Ok, I fixed it. The `requirements.txt` indeed yields problems with python 3.9. With python 3.8 it works alright.
For python 3.9 to push through, `matplotlib` needs to be removed from `requirements.txt`. As far as I could see, it is pulled in through a dependency anyhow.

An alternative `requirements.txt` I found helpful would be the following:
```
scikit-image
torch>=1.8
torchvision
nuscenes-devkit>=1.1.9
pillow
```
The above works with plain `pip` in python `3.8` and `3.9` while being less restrictive to what is currently coming with this repo.
